kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/RequestFilterResultBuilder.java

```diff
@@ -44,4 +44,15 @@ public interface RequestFilterResultBuilder extends FilterResultBuilder<RequestH
      */
     CloseOrTerminalStage<RequestFilterResult> shortCircuitResponse(@NonNull ApiMessage message) throws IllegalArgumentException;
 
+    /**
+     * Generate a short-circuit error response towards the client.
+     *
+     * @param header the headers from the request
+     * @param message the api message to generate an error in response too.
+     * @param throwable the exception that triggered the error response. Note Kafka will map the exception to an {@see org.apache.kafka.common.requests.ApiError} so callers may wish to supply choose their exception to trigger the appropriate error code
```

Comment:
should maybe link off to `Errors#forException` too since it's got the logic that will eventually default to `UNKNOWN_SERVER_ERROR`

Reply:
fixed in b882b4443e931e8c27434638a5bf8da079549406